src/components/Comments.js

```diff
@@ -1,21 +1,36 @@
 import React, { Component } from 'react';
 import CommentForm from './CommentForm';
-import { deleteComment } from '../redux/actions/blogActions';
+import EditComment from './EditComment';
+import { deleteComment, updateComment } from '../redux/actions/blogActions';
 import { connect } from 'react-redux';
 import '../css/comments.css';
 
 class Comments extends Component {
   state = {
-    showComments: false
+    showComments: false,
+    isEditCommentModeOn: false,
+    currentCommentId: ''
   };
   onShowClick = () => {
     this.setState({ showComments: !this.state.showComments });
   };
-
   onDeleteClick = (commentUuid, entryUuid, userId) => {
     this.props.deleteComment(commentUuid, entryUuid, userId);
   };
 
+  onEditCommentClick = (e, commentUuid) => {
+    if (e.target.id === commentUuid) {
+      this.setState({ isEditCommentModeOn: true, currentCommentId: e.target.id, hideEditAndDeleteButtons: true });
```

Comment:
This flag `hideEditAndDeleteButtons` is useless I think. If `isEditCommentModeOn` is true, the buttons should be hidden.